client/src/components/application/Examine/ActionButtons.vue

```diff
@@ -184,12 +184,8 @@ export default {
       // if there is no expiry date, this NR is not approved-expired
       if (this.$store.getters.expiryDate == null) return false;
 
-      let expired_date = moment(this.$store.state.expiryDate, 'YYYY-MM-DD').clone()
-      let today = new moment()
+      if (this.$store.getters.currentState === 'EXPIRED') return true
 
-      if (['APPROVED', 'CONDITIONAL'].includes(this.$store.getters.currentState) && today.isAfter(expired_date)) {
-        return true
-      }
       return false
```

Comment:
You have removed the check for approved statuses. This function will now only return "true for approved" if there is an expiry date and the status is not Expired. Is this OK?

I'd like to see the comment on line 184 updated. Maybe use a whole-function comment explaining what conditions mean what. (So a future developer doesn't have to guess the business logic.) Thx.

Reply:
Comments added